deepfence_server/reporters/scan/scan_result_actions.go

```diff
@@ -61,6 +61,15 @@ func UpdateScanResultMasked(ctx context.Context, req *model.ScanResultsMaskReque
 		return err
 	}
 	defer tx.Close()
+	getImageStubQuery := func() string {
+		imageStubQuery := ""
+		if value == true {
+			imageStubQuery = `MERGE (ist) -[ma:MASKED]-> (n) SET ma.masked = true`
+		} else {
+			imageStubQuery = `MATCH(ist) -[ma:MASKED] -> (n) DELETE ma`
+		}
+		return imageStubQuery
+	}
```

Comment:
Let's make this clearer by starting the query from:
```
MATCH (c) -[:IS] ->(ist)
```
Otherwise it will be hard to follow

Reply:
Removed the local function and moved the code inline.